### ISSUE TITLE
do not protect spawns and RC/OM in /devmap

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#2414]
In devmap, there is no reason to protect buildings against players mistakes.

**Comment by slipher** [comment]
`g_instantBuilding` enables cheat mode for building

**Comment by bmorel** [comment]
Not really, g_instantBuilding can be active even in normal mode, and it's even stored in config (this happened to me more than once, and it's annoying). Or maybe what is needed is a command to group several settings for actual dev mode, instead of only having to remember to type in several individual commands?

**Comment by slipher** [comment]
What do you mean "not really"?

It's a different setting because it messes with the normal gameplay mechanics. Sometimes when you are developing, the building stuff is what you want to test, so you need cheats for it to be off.

**Comment by bmorel** [comment]
To me g_instantBuilding is a different thing, even if it is true that it embeds the protection removal.
Using it in a playable (opposed to a dev) server could even be a thing, and then it would remove this protection, which I don't think is such a nice idea

**Comment by DolceTriade** [comment]
imo, I dislike the concept of using commands to modify persistent state (especially reversible state...). I consider devmap to be an exception, but even devmap is theoretically reversible by setting `g_cheats 0`.

**Comment by bmorel** [comment]
hm... `/devmap` is not  persistent AFAIK?

**Comment by bmorel** [comment]
but is not setting cvars persistent too?

**Comment by DolceTriade** [comment]
Yes, I think using cvars is a better way to express that state rather than commands.
